packages/mongodb-access-adapter/datagrip-access-adapter/src/main/kotlin/com/mongodb/jbplugin/accessadapter/datagrip/adapter/DataGripMongoDbDriver.kt

```diff
@@ -102,9 +109,11 @@ internal class DataGripMongoDbDriver(
     ): T? =
         withContext(Dispatchers.IO) {
             runQuery(
-                """db.getSiblingDB("${namespace.database.encodeForJs()}")
-                 .getCollection("${namespace.collection.encodeForJs()}")
-                 .findOne(EJSON.parse("${query.toJson()}"), EJSON.parse("${options.toJson()}")) 
+                """EJSON.serialize(
```

Comment:
Should these use canonical EJSON instead of relaxed?

Reply:
Ah true, I misread the configuration and the default configuration is relaxed, not canonical. Fixing this!